source/backend/apimodels/Models/Concepts/Property/HistoricalFileNumberMap.cs

```diff
@@ -1,33 +1,32 @@
-
 using Mapster;
 using Pims.Api.Models.Base;
 using Entity = Pims.Dal.Entities;
 
 namespace Pims.Api.Models.Concepts.Property
 {
-    public class HistoricalNumberMap : IRegister
+    public class HistoricalFileNumberMap : IRegister
     {
         public void Register(TypeAdapterConfig config)
         {
-            config.NewConfig<Entity.PimsFileNumber, HistoricalNumberModel>()
+            config.NewConfig<Entity.PimsFileNumber, HistoricalFileNumberModel>()
                 .Map(dest => dest.Id, src => src.FileNumberId)
                 .Map(dest => dest.PropertyId, src => src.PropertyId)
                 .Map(dest => dest.Property, src => src.Property)
-                .Map(dest => dest.HistoricalNumber, src => src.FileNumber)
-                .Map(dest => dest.HistoricalNumberType, src => src.FileNumberTypeCodeNavigation)
-                .Map(dest => dest.OtherHistoricalNumberType, src => src.OtherFileNumberType)
+                .Map(dest => dest.FileNumberTypeCode, src => src.FileNumberTypeCodeNavigation)
+                .Map(dest => dest.FileNumber, src => src.FileNumber)
```

Comment:
I think I preferred this being HistoricalNumber and HistoricalNumberType
Calling these file numbers conflates these values with our PIMS file numbers.

Reply:
I think these got changed by latest DB scaffold so need to wait until Eduardo rebases his branch